packages/expo-cli/src/commands/eject/Eject.ts

```diff
@@ -309,6 +310,35 @@ async function createNativeProjectsFromTemplateAsync(projectRoot: string): Promi
     process.exit(1);
   }
 
+  /**
+   * Add metro config, or warn if metro config already exists. The developer will need to add the
+   * hashAssetFiles plugin manually.
+   */
+
+  let updatingMetroConfigStep = logNewSection('Adding Metro bundler configuration');
+  if (
+    fse.existsSync(path.join(projectRoot, 'metro.config.js')) ||
+    fse.existsSync(path.join(projectRoot, 'metro.config.json')) ||
+    pkg.metro
+  ) {
+    updatingMetroConfigStep.stopAndPersist({
+      symbol: '⚠️ ',
+      text: chalk.red('Metro bundler configuration not applied:'),
+    });
+    log.nested(
+      `- You will need to add the ${chalk.bold(
+        'hashAssetFiles'
+      )} plugin to your existing Metro configuration. ${terminalLink(
+        'Example.',
+        'https://github.com/expo/expo/blob/master/packages/expo-updates/README.md#metroconfigjs'
+      )}`
+    );
+    log.newLine();
+  } else {
+    fse.copySync(path.join(tempDir, 'metro.config.js'), path.join(projectRoot, 'metro.config.js'));
```

Comment:
we might want to re-organize this a bit so that if fse.copySync throws an error for some reason we show the "Metro bundler configuration not applied" msg